src/js/components/Layer/__tests__/__snapshots__/Layer-test.js.snap

```diff
@@ -1615,27 +1721,33 @@ exports[`Layer invoke onClickOutside when modal={false} and layer has target 1`]
 `;
 
 exports[`Layer invoke onClickOutside when modal={false} and layer has target 2`] = `
-"@media only screen and (max-width: 768px) {
-  .iwgsjY {
-    position: fixed;
-    width: 100%;
-    height: 100%;
-    min-height: 100vh;
-  }
-}
-@media only screen and (max-width: 768px) {
-  .hmMkth {
-    position: relative;
-  }
+".gdHdpu {
+  font-size: 18px;
+  line-height: 24px;
+  box-sizing: border-box;
+  -webkit-text-size-adjust: 100%;
+  -ms-text-size-adjust: 100%;
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+  background: transparent;
+  position: relative;
+  z-index: 20;
+  pointer-events: none;
+  outline: none;
+  position: fixed;
+  top: 0px;
+  left: 0px;
+  right: 0px;
+  bottom: 0px;
 }
 @media only screen and (max-width: 768px) {
-  .dNkgGB {
+  .dbmsMO {
     position: relative;
     max-height: none;
     max-width: none;
     border-radius: 0;
-    height: 100vh;
-    width: 100vw;
+    height: 100%;
```

Comment:
This is switching from `100vh` to `100%` -- is this desired? or from a previous change that might not have received a snapshot update?